src/configure-nilrt-snac

```diff
@@ -141,6 +141,29 @@ remove_niauth() {
 }
 
 
+configure_tmux() {
+	log INFO Configuring tmux to be the default shell for both SSH and local
+	log INFO    logins, with a timeout...
+	opkg update >/dev/null && opkg install tmux
+
+	cat <<EOF >/etc/tmux.conf
+# NILRT SNAC configuration tmux.conf. Do not hand-edit.
+
+set -g lock-command vlock
```

Comment:
We can upstream this `lock-command` config line to meta-nilrt.

That would mean adding a new directory [here](https://github.com/ni/meta-nilrt/tree/nilrt/master/next/recipes-extended) with files like

`files/tmux.conf`
```
set -g lock-command vlock
bind X lock-session
```

`tmux_3.%.bbappend`
```
FILESEXTRAPATHS:prepend := "${THISDIR}/files:"

SRC_URI += "\
    file://tmux.conf \
"

do_install:append() {
    install -d ${D}${sysconfdir}
    install -m 644 ${WORKDIR}/tmux.conf ${D}${sysconfdir}/tmux.conf
}

RDEPENDS:${PN}:append = " vlock"
```

Reply:
Done. See ni/meta-nilrt#718